x-pack/filebeat/input/etw/input.go

```diff
@@ -198,7 +192,38 @@ func (e *etwInput) Run(ctx input.Context, publisher stateless.Publisher) error {
 }
 
 // buildEvent builds the winlog object.
-func buildEvent(data map[string]any, h etw.EventHeader, session *etw.Session, cfg config) map[string]any {
+func buildEvent(data map[string]any, h etw.EventHeader, session *etw.Session, cfg config) beat.Event {
+
+	winlog := map[string]any{
+		"activity_guid": h.ActivityId.String(),
+		"channel":       fmt.Sprintf("%d", h.EventDescriptor.Channel),
+		"event_data":    data,
+		"flags":         fmt.Sprintf("%d", h.Flags),
+		"keywords":      fmt.Sprintf("%d", h.EventDescriptor.Keyword),
+		"opcode":        fmt.Sprintf("%d", h.EventDescriptor.Opcode),
+		"process_id":    fmt.Sprintf("%d", h.ProcessId),
+		"provider_guid": h.ProviderId.String(),
+		"session":       session.Name,
+		"task":          fmt.Sprintf("%d", h.EventDescriptor.Task),
+		"thread_id":     fmt.Sprintf("%d", h.ThreadId),
+		"version":       fmt.Sprintf("%d", h.EventDescriptor.Version),
+	}
+
+	// Include provider GUID if not available in event header
+	zeroGUID := "{00000000-0000-0000-0000-000000000000}"
+	if winlog["provider_guid"] == zeroGUID {
+		winlog["provider_guid"] = session.GUID.String()
+	}
+
+	// Define fields map with Windows data and ECS mapping
+	fields := mapstr.M{
+		"winlog":         winlog,
+		"event.code":     fmt.Sprintf("%d", h.EventDescriptor.Id),
+		"event.created":  time.Now(),
```

Comment:
I like to always to set the TZ to UTC for consistency (e.g. `time.Now().UTC()`). You never know where this data might get logged/formatted (like during debugging) and having it always be UTC is nice IMO.

Reply:
Do the same for `convertFileTimeToGoTime`.